java/pom.xml

```diff
@@ -36,8 +36,8 @@
     <dep.netty.version>4.1.48.Final</dep.netty.version>
     <dep.jackson.version>2.11.4</dep.jackson.version>
     <dep.hadoop.version>2.7.1</dep.hadoop.version>
-    <dep.fbs.version>1.9.0</dep.fbs.version>
-    <dep.flatc.version>1.9.0</dep.flatc.version>
+    <dep.fbs.version>1.12.0</dep.fbs.version>
+    <dep.flatc.version>1.12.0</dep.flatc.version>
```

Comment:
is this still used?

Reply:
The `dep.flatc.version` is not; good catch. (the `dep.fbs.version` is, however). I have force pushed the change to remove it.